content/docs/user-guide/gems/reference/multiplayer/multiplayer-debug/debug/desync-audit-trail.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Multiplayer Desync Audit Trail
+description: Analyze and Debug Multiplayer Desyncs
+linkTitle: Desync Audit Trail
+---
+
+When the Client and Server disagree on the value of a networked variable, a desync occurs. Desyncs can be extremely difficult to debug which is where the Multiplayer Desync Audit Trail can help.
+
+The Multiplayer Desync Audit Trail is a tool available with the Multiplayer Gem in Debug environments. It details all network desyncs that occur and categorizes network activity leading up to a desync to attempt to root cause the desync itself.
+
+![Audit Trail Overlay](/images/user-guide/gems/reference/multiplayer/audit_trail_default.png)
+
+The Audit Trail will capture a desync and the Input ID and Host Frame it occurred on. For each desync, it will list under it all captured activity leading up to the desync up to a CVAR defined limit.
```

Comment:
".. will capture a desync, the Input Id and Host Frame it occurred on. For every desync, the UI will list all the captured activity leading to the descync. The depth of this history can be controlled by a CVar." or similar.